src/pages/staticPages/tools/mutationMapper/MutationMapperTool.tsx

```diff
@@ -18,11 +24,19 @@ import MutationMapperToolStore from './MutationMapperToolStore';
 
 import AppConfig from 'appConfig';
 import { getOncoKbApiUrl } from 'shared/api/urls';
+import { getBrowserWindow } from 'cbioportal-frontend-commons';
+import { referenceGenome } from 'shared/lib/referenceGenomeUtils';
 
 interface IMutationMapperToolProps {
     routing: any;
 }
 
+const ONCOKB_URL = 'https://www.oncokb.org/';
```

Comment:
are these already available in urls.ts?

Reply:
@alisman seems we only have oncokb.org/api/v1